docs/index.md

```diff
@@ -75,3 +75,8 @@ ObjectId using new mongoose.
 - Changes in default config: istioTraceHeaders and headerPropagation are deprecated in favor of a generic propagatedHeaders
 - Methods kafka.send that was deprecated is removed. Use kafka.producer.send instead
 - Mongoose migrated to v7 which has breaking changes see [here](https://mongoosejs.com/docs/7.x/docs/migrating_to_7.html)
+
+### Migrating from orka  4.x to 5.x
```

Comment:
@kalyvasio this is still migrating from 3.x to 4.x as we haven't gone out of beta yet. Use one line for the kafka changes